civismlext/preprocessing.py

```diff
@@ -273,6 +312,9 @@ def fit(self, X, y=None):
             self._check_sentinels(X)
             self.levels_ = self._create_levels(X)
 
+        # optionally flag unexpanded columns with nans
+        self.unexpanded_nans = self._flag_unexpanded_nans(X)
```

Comment:
Since this is set during the `fit`, I think should it be `self.unexpanded_nans_`. (Or `self._unexpanded_nans` if you want to keep it as internal-only.)